src/main/resources/org/jenkinsci/plugins/Messages.properties

```diff
@@ -1,5 +1,5 @@
-vSphereCloudSlave.OfflineReason.ShuttingDown=Shutting down VSphere Cloud Slave
+vSphereCloudSlave.OfflineReason.ShuttingDown=Shutting down VSphere Cloud Agent 
```

Comment:
Please remove trailing whitespace.

Reply:
Removed trailing space in the src/main/resources/org/jenkinsci/plugins/Messages.properties file.